bears/python/PEP8NotebookBear.py

```diff
@@ -0,0 +1,130 @@
+import autopep8
+from copy import copy
+import nbformat
+
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.settings.Setting import typed_list
+
+
+class NotebookHandler:
+    """Utility class for Jupyter Notebook handling."""
+
+    def __init__(self, filename, file):
+        """
+        Constructs a notebook handler to deal with NotebookNode objects.
+        A NotebookNode is a dict-like node with attribute-access.
+
+        :param filename: The filename of the file.
+        :param file:     The file contents as string array.
+        """
+        self.filename = filename
+        self.file = file
+        self._file_nb = self.read_from_list(file)
+
+    @staticmethod
+    def read_from_list(file):
+        """
+        Reads a notebook from a string array and returns the NotebookNode
+        object.
+
+        :param file: The notebook file contents as JSON string array.
+        :return:     The notebook as NotebookNode.
+        """
+        return nbformat.reads(''.join(file), nbformat.NO_CONVERT)
+
+    def write_to_list(self, file_nb):
+        """
+        Writes a notebook to a string array.
+
+        :param file_nb: The notebook (as NotebookNode) to write.
+        :return:        The notebook as JSON string array.
+        """
+        file_lst = (nbformat.writes(file_nb, nbformat.NO_CONVERT)
+                            .splitlines(True))
+        # nbformat.writes does not add a final \n, we do this manually
+        # necessary for the diff because original file has it
+        if self.file[-1].endswith('\n') and not file_lst[-1].endswith('\n'):
+            file_lst[-1] += '\n'
+        return file_lst
+
+    @property
+    def file_nb(self):
```

Comment:
it returns the file in notebook-node-form and I used `@property` to make sure that one only receives a copy to work with and does not change the original version of the 
notebook, i.e. `_file_nb` which is created at initialization.

what is you opinion?

apart from that: do you think this is redundant or maybe `file` be treated the same (to be consistent)?